rails_event_store_active_record/lib/rails_event_store_active_record/event_repository.rb

```diff
@@ -69,9 +65,9 @@ def has_event?(event_id)
     end
 
     def last_stream_event(stream_name)
-      build_event_entity(
-        EventInStream.where(stream: stream_name).order('position DESC, id DESC').first
-      )
+      record = EventInStream.where(stream: stream_name).order('position DESC, id DESC').first
+      return nil unless record
+      build_event_instance(record)
```

Comment:
```
return nil unless record
build_event_instance(record)
```

I think that could be `record && build_event_instance(record)`

:) but that's a minor